### DISCUSSION TITLE
Small object detection

### DISCUSSION THREAD

**Discussion by CesareDavidePace** [ultralytics/yolov5#10083] [Q&A]
Hi, 
I would need help with regards to the detection of very small objects, we are talking about objects even 20x20 on images of 5000x3000.
My approach was to use the sahi framework to do sliding windows and to keep the native size. In particular, my dataset is made up of satellite images.
My questions are:
1) What model can I use to improve the detection of very small objects? I have seen many recommend the yoloP2 but I don't know what changes.
2) Is it worth doing pretraining with the coco or training from scratch? And are there models already trained for images from above?
3) I did sliding windows in 500x500, is it better to increase the tiles for example 1280x1280 since the networks are designed for that?

**Comment by Kanan99** [comment]
Hey,

How did you implement the custom model on SAHI? 

I would be glad if you share your experience :)
